events/class_tests/meeting_vote_tests.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+from events.forms import VoteForm
+from django.forms import formset_factory
+from main.utilities import send_invite_notification
+from pytest_django.asserts import assertTemplateUsed
+from events.models import PossibleParticipant, OptionalMeetingDates
+
+
+@pytest.mark.django_db
+class TestMeetingVote:
+
+    def test_create_valid_meeting_vote(self, get_event_participant, valid_meeting_vote_data):
```

Comment:
as far as i understand a new possible participant is been created after the vote as been,
if thats correct then why dont you save the vote form and then check that a new possible particpant been made?

Reply:
If you will see the vote form is only a Boolean field,
I wanted to show the user a checkbox style voting form and this is the way to do it with formset.
If I would create a form for meeting date it will not look like this and in this way there is no errors or dealing with unwanted situations